libhif/hy-query.c

```diff
@@ -1274,3 +1274,36 @@ hy_query_run_set(HyQuery q)
     hy_query_apply(q);
     return hif_packageset_from_bitmap(q->sack, q->result);
 }
+
+/**
```

Comment:
Could you use gtk-doc formatted headers here please. When I convert this to a GObject we'll need to make these all introspectable.